covid19_sfbayarea/data/alameda/utils.py

```diff
@@ -0,0 +1,9 @@
+from functools import reduce
+from typing import Any, Dict, List
+
+def dig(items: Dict[str, Any], json_path: List[Any]) -> Any:
+    try:
+        return reduce(lambda subitem, next_step: subitem[next_step], json_path, items)
+    except (KeyError, TypeError, IndexError) as err:
+        print('Error reading returned JSON, check path: ', err)
+        raise(err)
```

Comment:
Not a big deal, but this is generic enough that you might as well move it up one or two levels to `covid19_sfbayarea.data.utils` or `covid19_sfbayarea.utils`.

Reply:
Oh, just noticed a minor typing issue: if `items` is always keyed by strings (`Dict[str, Any]`), `json_path` should probably be restricted to a list of strings (`List[str]`).

Reply:
Oh this is a good point - `items` should be a `dict` but can contain a `list` at a deeper level, so `ints` are appropriate, so it should maybe be `List[Union[int, str]]`.

Reply:
Moved it to upper `utils` and I think this is correct typing now: https://github.com/sfbrigade/data-covid19-sfbayarea/pull/139/commits/e7b3520e0feda1899a7110b3259108a8103c96b8